docs/updating_schemas.md

```diff
@@ -236,6 +324,75 @@ To add an object with accession prefix 'SM':
+        * `linkTo`: A list of valid objects to link to. Must be the name of a schema, which can be abstract or concrete.
+        * `pattern`: A regex pattern applied to the string.
+    * Numbers or integers:
+        * `minimum`: A minimum value.
+        * `maximum`: A minimum value.
+    * Arrays:
+        * `items`: Required. A definition of the items that are valid in the array. Must include `type` at minimum.
+        * `minItems`: Required. An integer indicating the minimum number of items in an array. Must be at least 1.
+        * `maxItems`: An integer indicating the maximum number of items in an array.
+        * `uniqueItems`: Required. A boolean which is always `True`.
+    * Objects:
+        * `properties`: A nested schema. Must obey above requirements for each property: `title`, `type`, and `description`.
+
+The following are properties categorized by IGVF members' permission to submit. These properties must follow the preceding guidelines according to their `type` in addition to the guidelines below.
+* Submittable properties:
+    * `comment`: Required. Instructions for submitters explaining details of how to submit the property. If there is a dependency defined for the property, the comment must include: "This property is affected by dependencies. See \[link\]."
+    * `submissionExample`: Required. An example of how to specify the property using appscript and igvf_utils tools.
+* Admin only properties:
+    * `permission`: Required. A string property which is always `import_items`. Must not be present for any property that can be modified by any submitter.
+* Calculated properties:
+    * `notSubmittable`: Required. A boolean property which is always `True`. Must not be present for any submittable property.
+
+-----
+
+Changelog guidelines
+----------------
+
+* Do record changes in calculated properties.
+* Don't record changes to searches, such as fuzzy vs. exact, or to embedding.
+* Add entries to all affected schemas when there is a change to properties which are inherited or merged into other schemas, such as mixins.
+* Add new entries at the top of the changelog.
+* Record changes occurring in different PRs as separate entries.
+* Mention only one property per line. Additionally, only one enum should be mentioned in a line where possible. For example, if 10 enums are added to a property, each one is noted on a separate line.
+* Begin with a present tense verb describing the action: add, remove, rename. Adhere to the stylebook whenever possible.
+
+
+### Stylebook
+The following table provides common types of schema changes and provides a specific sentence pattern to follow for the entry.
+|Type of schema change|Sentence example|
+|-|-|
+|Add a property|Add \[property name\].|
+|Remove a property|Remove \[property name\].|
+|Replace a property|Rename \[old name\] to \[new name\].|
+|Add an enum|Extend \[property name\] enum list to include \[enum\].|
```

Comment:
are there two options for this?

Reply:
I'm a little confused on this comment, I think we only discussed one option for "adding an enum". But maybe I am forgetting?

Reply:
option 1) Adding an envm vs. 2) Extend enum list to include...

Reply:
this is resolved. This is a table format and I was reading this as `option1|option2`